Cython/Compiler/Nodes.py

```diff
@@ -3234,6 +3237,13 @@ def generate_argument_parsing_code(self, env, code):
         # Move arguments into closure if required
         def put_into_closure(entry):
             if entry.in_closure:
+                if entry.type.is_memoryviewslice:
+                    error(
+                        self.pos,
+                        "Putting capturing a memoryview argument in a closure is not supported. "
```

Comment:
"Putting capturing" is difficult to understand for me. "Capture" doesn't feel like a common term here.

Maybe: "Referring to a memoryview typed argument directly in a nested closure function is not supported in Cython 0.x."

Reply:
"Putting capturing" doesn't make any kind of sense to me either now that I re-read it.

I've change it to your proposed wording.